gensim/models/keyedvectors.py

```diff
@@ -215,7 +215,7 @@ def __str__(self):
 class BaseKeyedVectors(utils.SaveLoad):
     """Abstract base class / interface for various types of word vectors."""
     def __init__(self, vector_size):
-        self.vectors = zeros((0, vector_size))
+        self.vectors = zeros((0, vector_size), dtype=REAL)
```

Comment:
Is there a reason why we don't explicitly say `float32` here? What's the benefit of using `REAL` as an alias?

Reply:
idk, just "mimic" to existing code